lib/chef_backup/strategy/backup/tar.rb

```diff
@@ -129,6 +142,10 @@ def config_directories
     [project_name] + enabled_addons.keys
```

Comment:
Just curious, [down here](https://github.com/chef/chef_backup/pull/25/files#diff-7b185048daf6de8237d6e46f76bd37c4R146) you've guarded against duplicate entries, but for `config_directories` this doesn't matter?